src/cli/index.js

```diff
@@ -4,22 +4,46 @@ const path = require('path');
 
 const fse = require('fs-extra');
 const isEmpty = require('lodash/isEmpty');
+const isString = require('lodash/isString');
 const camelCase = require('lodash/camelCase');
 const lowerCase = require('lodash/lowerCase');
 const { printMessage, formatMessage } = require('formatted-messages');
 
+const isEmptyString = s => isString(s) && isEmpty(s);
```

Comment:
This keeps being duplicated in each file. It could probably just be done once in utility or made into a separate tiny NPM package 🤷‍♂. I think I thought it would be nicer to be in the same file for clarity, however, it's increasing the chances that I mess it up just once and introduce a bug.

Reply:
Moved into utility 👍